practice-app/website/api/interesteds.py

```diff
@@ -9,7 +9,7 @@
 # endpoint for declaring a person as interested to an event
 # event is determined by the route, user should be passed as a paremeter
 @interesteds.route('/<event_id>/interesteds', methods = ['POST'])
-@swag_from('doc/interesteds_post.yml', methods = ['POST'])
+@swag_from('doc/interesteds_POST.yml', methods = ['POST'])
```

Comment:
Writing "post" with capital letters is better in terms of readability and clearness. Nice change.